src/cloud/enrichments/README.md

```diff
@@ -81,6 +81,21 @@ This file contains all the mapped gaze data from all sections. The coordinate sy
 | **gaze position on surface x [normalized]** | Float value representing the x-coordinate of the mapped gaze point in surface coordinates. If the surface was not localized this value is empty.     |
 | **gaze position on surface y [normalized]** | Same as gaze position on surface x [normalized] but for y-coordinate.     |
 
+#### fixations.csv
+This file contains fixation events detected in the gaze data stream and mapped to the surface.
+
+
+| Field | Description | 
+| -------- | -------- | 
+| **section id** | Unique identifier of the corresponding section.     |
+| **recording id** | Unique identifier of the recording this sample belongs to.     |
+| **fixation index** | Index of fixation within the section. The index corresponds to the fixation index of the raw unmapped data.    |
```

Comment:
I think we should rename this to `fixation id` to be consistent with other columns and since within a section the fixation may not start at 1 which can cause confusion.

Reply:
Makes sense to me!